providers/implementations/ciphers/cipher_aes_wrp.c

```diff
@@ -212,12 +212,12 @@ static int aes_wrap_cipher(void *vctx,
 
     if (outsize < inl) {
         ERR_raise(ERR_LIB_PROV, PROV_R_OUTPUT_BUFFER_TOO_SMALL);
-        return -1;
+        return 0;
     }
 
     len = aes_wrap_cipher_internal(ctx, out, in, inl);
-    if (len == 0)
-        return -1;
+    if (len <= 0)
+        return 0;
```

Comment:
Err did you mean to include the change in this file?

Reply:
Uhm, nope!

Reply:
Gone now